WebApplication1/Startup.cs

```diff
@@ -38,6 +38,8 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                 app.UseExceptionHandler("/Error");
                 // The default HSTS value is 30 days. You may want to change this for production scenarios, see https://aka.ms/aspnetcore-hsts.
                 app.UseHsts();
+                username=rahyul@gmail.com
+                    password=125488ghtyun
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'password'.](https://app.codacy.com/app/rahulroyyadav95/ASP_BASIC/pullRequest?prid=4638650)